contracts/hardhat.config.ts

```diff
@@ -42,25 +42,29 @@ const config: HardhatUserConfig = {
         accountPath: '~/.starknet_accounts',
       },
     },
+    requestTimeout: 1000000,
   },
   networks: {
     devnet: {
       url: 'http://127.0.0.1:5050',
+      args: ['--cairo-compiler-manifest', '../vendor/cairo/Cargo.toml'],
     },
     integratedDevnet: {
       url: 'http://127.0.0.1:5050',
       venv: 'active',
-      args: ['--lite-mode'],
+      args: ['--lite-mode', '--cairo-compiler-manifest', '../vendor/cairo/Cargo.toml'],
```

Comment:
Is this still necessary? I was hoping to remove the `vendor/` dirs later on since we can use binary artifacts

Reply:
I guess it's necessary until devnet supports the latest rc0